### PR TITLE
Add UpdateURL to graphql schema and resolvers

### PR DESCRIPTION
**Pull request by oatovar**
* Update GraphQL Schema's AuthMutation to include `updateURL`.

* Refactor to use single `UrlInput` in both `updateURL` and `createURL`.

* Regenerate wired dependencies.

### CODE REVIEW

File: backend/app/adapter/graphql/schema.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ type AuthQuery {
 
 type AuthMutation {
 	createURL(url: URLInput!, isPublic: Boolean!): URL
+	updateURL(alias: String!, url: URLInput!, isPublic: Boolean): URL
```
Comment (byliuyang) [inline backend/app/adapter/graphql/schema.go:23]: We'll have to discuss the `isPublic` parameter in the future.

**Comment by byliuyang** [comment]
Hi @oatovar , thank you very much for working on this feature! I think you need to add GraphQL schema with resolvers. Please resolve PR conflicts!

**Comment by oatovar** [comment]
@byliuyang thank you for the feedback. I will take a look at the resolvers and fix the PR conflicts.

**Comment by byliuyang** [comment]
@oatovar I recommend we open PRs in the direction of dependency graph:
image

1. Repositories
2. Use cases
3. GraphQL resolvers & scheme

**Comment by oatovar** [comment]
@byliuyang I went with the solution that you recommend in #239. Please let me know where I can improve. Thank you!

**Comment by oatovar** [comment]
Closing and making new PR.

**Pull request closed without merging**